### PR TITLE
Prince/ POC new crowdin flow

### PR DESCRIPTION
**Pull request by prince-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: scripts/extract-new-texts.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          for (let i = 0; i < globs.length; i++) {
+            let filesFound = glob.sync(`../src/${globs[i]}`);
+            filesFound = filesFound.filter(
+              (path) => path.indexOf("__tests__") === -1
+            );
+            file_paths.push(...filesFound);
+          }
+          // Iterate over files and extract all strings from the i18n marker
+          for (let i = 0; i < file_paths.length; i++) {
+            if (program.verbose) {
```
Comment (habib-deriv) [inline scripts/extract-new-texts.js:59]: We don't need this, right?

File: scripts/extract-new-texts.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            } catch (e) {
+              console.log(e);
+            }
+          }
+
+          // Localization File Content Generator
+          let locale_content = `/* eslint-disable  */ \n  new_texts = [\n`
+
+          // Log to avoid duplicates in untranslated list
+          const unstranslated = []
```
Comment (habib-deriv) [inline scripts/extract-new-texts.js:80]: typo

File: scripts/extract-new-texts.js (hunk 1 above)
Comment (prince-deriv) [reply scripts/extract-new-texts.js:59]: This is needed. We need to get all the text to be translated this is a copy of extractor we have but slightly modified to achieved our requirements

File: scripts/extract-new-texts.js (hunk 2 above)
Comment (prince-deriv) [reply scripts/extract-new-texts.js]: done 👍🏼

**Pull request closed without merging**